ConsistencyManagerTests/ListenerTests.swift

```diff
@@ -105,6 +105,7 @@ class ListenerTests: ConsistencyManagerTestCase {
                     listener = strongListener
 
                     addListener(strongListener, toConsistencyManager: consistencyManager)
+                    waitOnMainThread()
```

Comment:
It waits until all blocks are finished. Does a dispatch_async and blocks until it runs.
```waitForMainThreadToFlush()```
```waitForMainThreadTasksToComplete()```
```waitUntilMainThreadWorkFinished()```

ideas ^^^? TBH, I'm not a huge fan of any of them for verbosity...but can see your point here.